session.go

```diff
@@ -160,3 +160,30 @@ func (s *session) sendCommand(cmdType command, cmd string) Packet {
 	results := s.readNBytes(int(r1))
 	return newPacket(int(rep), results)
 }
+
+func (s *session) sendvoidCommand(cmdType command, cmd string) error {
```

Comment:
to avoid duplicate codes, introduced exeCommand function, inlcuding preaprecmd() and send out command bytes.
I will submit a new request.